fhirstore/search/search_methods.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import re
+import json
+
+from collections import defaultdict
+from elasticsearch import Elasticsearch
+
+number_prefix_matching = {"gt": "gt", "ge": "gte", "lt": "lt", "le": "lte"}
+
+def build_element_query(key, value):
+    """ Translates the a single JSON body search to an
+    elasticSearch query
+    """
+
+    element_query = defaultdict(lambda: defaultdict(dict))
+
+    numeric_modif = re.search(r"^(gt|lt|ge|le)([0-9].*)$", f"{value}")
+    eq_modif = re.search(r"^(eq)([0-9].*)$", f"{value}")
+    str_modif = re.search(
+        r"^(.*):(contains|exact|above|below|not|in|not-in|of-type)$", key
+    )
+    if str_modif:
+        if str_modif.group(2) == "contains":
```

Comment:
could you name two variables for str_modif.group(1) and str_modif.group(2) to improve readability?
Also, use if / elif to avoid to do all checks and improve efficiency

Reply:
Ok, I will rename those